test/utils/FakeObjects.scala

```diff
@@ -57,4 +59,112 @@ trait FakeObjects {
     uploadedFiles = Seq(evidence),
     submissionSource = "DFE_UI"
   )
+
+
+  val createValidRequest: GroupAccountSubmission = GroupAccountSubmission(
```

Comment:
can we call this something related to group account submission, while in fake objects there could be many create valid requests, such as property link create.